app/controllers/recipes_controller.rb

```diff
@@ -19,5 +19,7 @@ def destroy
     redirect_to root_path, notice: 'Successfully deleted recipe.' if Recipe.destroy(params[:id])
   end
 
-  def public_recipes; end
+  def public_recipes
+    @recipes = Recipe.where(public: true).order('created_at Desc')
+  end
```

Comment:
[Optional] In this line number 23 kindly replace `@recipes = Recipe.where(public: true).order('created_at Desc')` to `@recipes = Recipe.where(public: true).order(created: :desc)`